### PR TITLE
sensor: move timer configuration to Kconfig

### PR DESCRIPTION
**Pull request by RobertGalatNordic**
KRKNWK-16974
Signed-off-by: Robert Gałat <robert.galat@nordicsemi.no>

### CODE REVIEW

File: samples/sensor_monitoring/Kconfig [hunk 1]
```diff
@@ -3,6 +3,48 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+
+config DEMO_CAPABILITY_PERIOD_MS
+        int
```
Comment (ktaborowski) [inline samples/sensor_monitoring/Kconfig:8]: No prompt?

File: samples/sensor_monitoring/Kconfig [hunk 2]
```diff
@@ -3,6 +3,48 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+
+config DEMO_CAPABILITY_PERIOD_MS
```
Comment (ktaborowski) [inline samples/sensor_monitoring/Kconfig:7]: Consider some common prefix e.g. `SM_TIMER_`

File: samples/sensor_monitoring/Kconfig [hunk 3]
```diff
@@ -3,6 +3,48 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
+
```
Comment (ktaborowski) [inline samples/sensor_monitoring/Kconfig:6]: Consider a menu `SM_TIMERS` to better organize configs

File: samples/sensor_monitoring/Kconfig (hunk 1 above)
Comment (RobertGalatNordic) [reply samples/sensor_monitoring/Kconfig:11]: added prompt

File: samples/sensor_monitoring/Kconfig (hunk 2 above)
Comment (RobertGalatNordic) [reply samples/sensor_monitoring/Kconfig]: added prefix

**Review by ktaborowski — APPROVED** [review]
New configs works fine

**Pull request merged**